tests/Adapter/ReferenceFinder/IndexedReferenceFinderTest.php

```diff
@@ -102,6 +103,54 @@ class Bar extends Foo {}
         ];
     }
 
+    
+    /**
+     * @return Generator<mixed>
+     */
+    public function provideTraits(): Generator
+    {
+        yield 'single trait' => [
+            <<<'EOT'
+// File: project/subject.php
+<?php class Foo { use Ba<>r; };
+EOT
+            ,
+            1
+        ];
+
+        yield 'implementation' => [
+            <<<'EOT'
+// File: project/subject.php
+<?php class Foo { use Ba<>r; };
+
+// File: project/other.php
+<?php
+$b = new Foo();
+EOT
+            ,
+            2
+        ];
+
+        // TODO: This fails
+//         yield 'implementation (method)' => [
+//             <<<'EOT'
+// // File: project/subject.php
+// <?php trait Bar { function f<>unc(); }
+
+// // File: project/foo.php
+// <?php class Foo { use Bar; };
+
+// // File: project/other.php
+
+// <?php
+// $b = new Foo();
+// $b->func();
+// EOT
```

Comment:
please remove all commented code (in general no TODOs, but especially not commented code)

Reply:
Ye, I keep forgetting those. Sorry about that.

Reply:
Actually I didn't forget it - I meant to leave it as a failing test should someone decide to go after it.